src/service/resources-impl.js

```diff
@@ -385,6 +385,19 @@ export class Resources {
   }
 
   /**
+   * Invokes `unload` on the elements' resource which in turn will invoke
+   * the `documentBecameInactive` callback on the custom element.
+   * @param {!Element} parentElement
+   * @param {!Element|!Array<!Element>} subElements
+   */
+  schedulePause(unusedParentElement, subElements) {
+    elements_(subElements).forEach(el => {
+      const r = this.getResourceForElement(el);
```

Comment:
This, unfortunately, won't work. There's no guarantee that any of the subElements (in our cases, these are slides) are actually AMP elements - it could simply be a div. You need to use `discoverResourcesForArray_` like in `scheduleLayoutOrPreloadForSubresources_`. That's why `parentElement` is actually important and should be "used" :)